tests/acceptance/infinity-route-offset-trigger-test.js

```diff
@@ -51,8 +52,8 @@ function infinityLoader() {
 }
 
 function triggerOffset() {
-  var list = postList();
-  return list.get(0).scrollHeight - list.height();
+  let { top } = document.getElementsByClassName('infinity-loader')[0].getBoundingClientRect()
+  return top - Ember.$(window).height();
```

Comment:
This is essentially what ember-in-viewport is doing.

Reply:
And it allows the tests to pass.  Some change in the ember-testing container is causing pain for our test setup.  This is what ember-cli-qunit is applying to `#ember-testing`

```
height: 200%;
width: 200%;
transform: scale(0.5);
```

So the way we were testing it before wasn't playing nicely with the above css.